src/pocketmine/Player.php

```diff
@@ -3241,7 +3241,14 @@ public function handleItemFrameDropItem(ItemFrameDropItemPacket $packet) : bool{
 
 		$tile = $this->level->getTile($this->temporalVector->setComponents($packet->x, $packet->y, $packet->z));
 		if($tile instanceof ItemFrame){
+			$ev = new PlayerInteractEvent($this, $this->inventory->getItemInHand(), $tile->getBlock(), $tile->getBlock()->getDamage(), PlayerInteractEvent::LEFT_CLICK_BLOCK);
```

Comment:
I'm not sure if LEFT_CLICK_BLOCK is appropriate. Usually, LEFT_CLICK_BLOCK implies start breaking a block, which may lead to confusion.